ecsclient/baseclient.py

```diff
@@ -71,6 +72,7 @@ def __init__(self, username=None, password=None, token=None,
             token_path=self.token_path,
             request_timeout=self.request_timeout,
             cache_token=self.cache_token)
+        self.override_header =override_header 
```

Comment:
why is this indented weird?

Reply:
if you expand and see the top code, it is not indented weird. However, I moved the code other place now.